extension/src/lib/zoom.js

```diff
@@ -0,0 +1,50 @@
+/*globals fluid */
+
+"use strict";
+
+var gpii = fluid.registerNamespace("gpii");
+var chrome = chrome || require("sinon-chrome");
+
+fluid.defaults("gpii.chrome.zoom", {
+    gradeNames: "fluid.modelComponent",
+    model: {
+        magnifierEnabled: false,
+        magnification: 1
+    },
+    invokers: {
+        set: {
+            funcName: "gpii.chrome.zoom.set",
+            args: "{arguments}.0"
+        }
+    },
+    modelListeners: {
+        "*": {
```

Comment:
Nope, I also want to listen to changes on "magnification". This is mainly to take care of updates in the current personalization triggered by a context change. i.e.: 
* I'm already logged in and I'm already using magnification
* At 10 pm I usually need a little bit more of magnification, so I've got a context that describes these preferences.